src/main/java/hudson/plugins/jira/JiraCreateIssueNotifier.java

```diff
@@ -329,16 +339,18 @@ private void currentBuildResultFailure(AbstractBuild<?, ?> build, BuildListener
             
             //Get the issue-id which was filed when the previous built failed
             String issueId = getIssue(filename);
-            if (issueId != null) {
-                try {
-                    //The status of the issue which was filed when the previous build failed
-                    Status status = getStatus(build, issueId);
+            Status status = getStatus(build, issueId);
```

Comment:
This fix stays here as is. Reason being, say for some reason we have the need to use the `forceNewIssue` as false, we don't want to be getting the error we are getting right now, specifically FOT-4716.
For this and the other comment, there are 2 ways to go about it.
1. We rework the current `perform` call to do what we want it to do ie. the current state of the PR
or
2. We overload `currentBuildResultFailure` and call it only when `forceNewIssue` is true. That way we shortcut all of that excess logic and basically use the second half of that function starting from line 368. 
This means adding test code and basically spending more time on it.